ceph_installer/templates.py

```diff
@@ -19,6 +19,10 @@
 fi
 
 
+echo "--> Installing Python for Ansible"
+sudo apt-get update
```

Comment:
`sudo` is not needed as the user is required to call this script with super user privileges